server/db/index.js

```diff
@@ -18,12 +18,12 @@ const syncAndSeed = () => {
         Promise.all(challengesSeed.map(chal => Challenge.create(chal))),
         Promise.all(userchallengeSeed.map(chal => Userchallenge.create(chal))),
         Promise.all(solutionsSeed.map(sol => Solution.create(sol))),
-        Promise.all(imagesSeed.map(img => Image.create(img))),
+        Promise.all(
+          imagesSeed.map(img => Image.create({ connector: img.connector, data: img.data })),
+        ),
       ]);
     })
     .then(([users, challenges, userchallenges, solutions, images]) => {
```

Comment:
users and challenges are not used in the code below, how about in stead of using hard code number 3, 2 on line 33, we use sth like users[Math.floor(Math.random() * 10)].id and challenges[Math.floor(Math.random() * 10)].id